numpy/f2py/capi_maps.py

```diff
@@ -22,7 +22,7 @@
 from .crackfortran import markoutercomma
 from . import cb_rules
 
-# The environment provided by auxfuncs.py is needed for some calls to eval.
+# The eviroment provided by auxfuncs.py is needed for some calls to eval.
```

Comment:
Looks like your rebase has gone wrong, this reverts the spelling mistakes you fixed!

Reply:
This comment still applies

Reply:
```suggestion
# The environment provided by auxfuncs.py is needed for some calls to eval.
```